tools/cluster/tests/wasp-cli_test.go

```diff
@@ -139,6 +139,17 @@ func TestWaspCLIContract(t *testing.T) {
 	checkCounter(45)
 }
 
+func findRequestIDInOutput(out []string) string {
+	for _, line := range out {
+		m := regexp.MustCompile(`(?m)#\d+ \(check result with: wasp-cli chain request (\w+)\)$`).FindStringSubmatch(line)
+		if len(m) == 0 {
+			continue
+		}
+		return m[1]
+	}
+	return ""
+}
```

Comment:
regexp could be created once and re-used

```suggestion
var rxp = regexp.MustCompile(`(?m)#\d+ \(check result with: wasp-cli chain request (\w+)\)$`)
func findRequestIDInOutput(out []string) string {
	for _, line := range out {
		m := rxp.FindStringSubmatch(line)
		if len(m) == 0 {
			continue
		}
		return m[1]
	}
	return ""
}
```

Reply:
True, but this being test code I'd argue that the performance gain is not important. Personally I prefer to leave it as it is.